tests/tacacs/test_accounting.py

```diff
@@ -27,19 +31,25 @@ def cleanup_tacacs_log(ptfhost, rw_user_client):
 
     ssh_run_command(rw_user_client, 'sudo truncate -s 0 /var/log/syslog')
 
+
 def check_tacacs_server_log_exist(ptfhost, tacacs_creds, command):
     username = tacacs_creds['tacacs_rw_user']
     """
         Find logs run by tacacs_rw_user from tac_plus.acct:
             Find logs match following format: "tacacs_rw_user ... cmd=command"
             Print matched logs with /P command.
     """
+
+    # On some hardware, syslog write to log file very slow, wait 3 seconds untill log file ready.
+    time.sleep(3)
```

Comment:
Most code format change in this PR is because the per-commit-check failed.

Only the 2 sleeps at line 44 and line 76 are code change.